### ISSUE TITLE
sugestão: adicionar código município TSE

### ISSUE THREAD

**Issue by sfinotti** [kelvins/Municipios-Brasileiros#47]
Primeiramente não consigo entender por qual razão o TSE não usa a codificação de municípios do IBGE. 
Dito isto e vendo necessidade constante de trazer dados de coordenadas para as informações do TSE, sugiro acréscimo de uma coluna "cd_mun_TSE".

**Comment by matheus0sa** [comment]
Achei um repositório, que inclusive foi inspirado nesse, com os municípios, os códigos do IBGE e do TSE, seria um join dessas tabelas

https://github.com/betafcc/Municipios-Brasileiros-TSE/tree/mastera

**Comment by kelvins** [comment]
Boa, acho que seria muito útil. Se alguém tiver tempo e interesse em abrir o PR ficaria feliz em revisar. Minha sugestão seria utilizar o mesmo nome de coluna utilizado pelo @betafcc (`codigo_tse`).

**Comment by sfinotti** [comment]
> Achei um repositório, que inclusive foi inspirado nesse, com os municípios, os códigos do IBGE e do TSE, seria um join dessas tabelas
> 
> https://github.com/betafcc/Municipios-Brasileiros-TSE/tree/mastera

Sim, eu tenho usado esta base que vc mencionou nos meus projetos. Único detalhe é que ela tem 5 anos que tem atualizações.

**Comment by dgansauskas** [comment]
Oi pessoal tudo bem?

Acabei encontrando esse repo excelente do @kelvins <3 durante minhas pesquisas para criar uma tabela unificada que traz alguns dados de municípios e estados brasileiros atualizados e, vendo essa thread, eu gostaria de sugerir que fossem extraídos dados direto do TSE:
https://dadosabertos.tse.jus.br/dataset/eleitorado-2022/resource/8a6ff7bd-5a22-4d1a-9353-cce06f9f5e35

No caso, esse link já deva direto para o download da base mais atual, que é a do ano de 2022.

Vou abrir uma outra issue para saber se seria possivel adicionarmos outra informação nessa base.

Falando nisso, @kelvins, vi que vc tem algumas issues abertas e gostaria de me dispor aqui pra ver se vc precisa de alguma ajuda na atualização e manutenção desse repo. Se sim, fico à disposição!

**Comment by kelvins** [comment]
@dgansauskas mil desculpas pela demora em responder, minha vida está uma loucura nesses últimos meses.

Então, já faz um tempo que não utilizo esses dados em um projeto real então não estou ativamente dando manutenção no projeto mas estou disposto em continuar revisando PRs e contribuindo com as discussões.

Sinta-se mais do que bem vindo em contribuir e auxiliar na manutenção do repositório ❤️ 
Caso tenha interesse posso te adicionar como colaborador do projeto.
